onnx/reference/op_run.py

```diff
@@ -403,6 +403,40 @@ def _run(self, *args, **kwargs):  # type: ignore
             f"Method '_run' must be overwritten for operator {self.__class__.__name__!r}."
         )
 
+    def _check_output_run(self, res: tuple[Any]) -> tuple[Any]:
+        """
+        Checks the output are from the expected type.
+        """
+        if not isinstance(res, tuple):
+            raise TypeError(
+                f"Method '_run' of class {self.__class__.__name__!r} does not return a tuple but {type(res)}."
```

Comment:
```suggestion
                f"Method '_run' of class {self.__class__.__name__!r} does not return a tuple but returns '{type(res)}'."
```

Reply:
I usually add quotes if it is a string (`!r` is handling it). Is there any rule for exception style?

Reply:
No in particular, but I have seen when a type is complex sometimes it get blurred with the text without quoting. I usually quote all types in error messages. This tends to be consistent with python's type error:

```
>>> foo = ""
>>> foo()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'str' object is not callable
```

Reply:
done